serverless.yaml

```diff
@@ -145,6 +127,7 @@ functions:
 
 resources:
   - Conditions:
+      isNotProd: !Not !Equals ['${self:custom.stage}', 'prod']
       isDev: !Equals ['${self:custom.stage}', 'dev']

```

Comment:
Why do we have both `isNotProd` and `isDev`?

Reply:
My distinction is we only want the expensive elastic search if the users use stage 'prod'. perhaps better variable name would help?

Reply:
What was wrong with using `isDev` to keep ES small?

I think we should be careful about supporting more "special stage names" like 'dev', 'prod'. The fewer the better.

Also, maybe we should not do make this change as part of this PR anyways. It's not related to SMART and Idk if it will reach mainline given the branching strategy for SMART.

Reply:
Good point encapsulating PRs is a best practice - reverting